base/error.go

```diff
@@ -31,12 +32,15 @@ type SGError struct {
 
 var (
 	ErrAlreadyImported = &SGError{alreadyImported}
+	ErrImportCancelled = &SGError{importCancelled}
 )
 
 func (e SGError) Error() string {
 	switch e.code {
 	case alreadyImported:
 		return "Document already imported"
+	case importCancelled:
```

Comment:
ImportCancelled is used when an import operation doesn't need to be completed for a given SDK update.  It won't show up in the logs as a warning or error. 

When Import+ is enabled, we're already including verbose logging about the nature of the scenario (below).